templates/zerver/integrations.html

```diff
@@ -1244,6 +1244,49 @@
 
     </div>
 
+    <div id="mention" class="integration-instructions">
+
+      <p>First, create the stream you'd like to use for Mention Webhook notifications,
+      and subscribe all interested parties to this stream. The integration will
+      use the default stream <code>mention</code> if no stream is supplied in
+      the hook; you still need to create the stream even if you are using this
+      default.</p>
+
+      <p>Next, go to your feed and click on your profile in the top right corner,
```

Comment:
You should clarify that you are talking about Mention profile and not Zulip profile.